sanitizer/_text_pelias_parser.js

```diff
@@ -100,33 +142,31 @@ function parse (clean) {
     .map((c, i) => (mask[i] !== 'P') ? c : ' ')
     .join('');
 
+  // same as $body above but with consecutive whitespace squashed and trimmed.
+  const normalizedBody = t.section.map(sp => sp.body).join(' ').replace(/\s+/g, ' ').trim();
```

Comment:
This is not quite the same as above since it includes `postalcode` classifications.
Need to give this some more consideration before merging 🤔